docs/DeveloperGuide.md

```diff
@@ -375,18 +377,20 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 1.  Should work on any _mainstream OS_ as long as it has Java `11` or above installed.
 2.  Should be able to hold up to 1000 items without a noticeable sluggishness in performance for typical usage.
 3.  A user with above average typing speed for regular English text (i.e. not code, not system admin commands) should be able to accomplish most of the tasks faster using commands than using the mouse.
-4.  Project should not cost any money
-5.  Should work on 32-bit and 64-bit environments
-6.  Should not take up more than 50 MB of disk space
-7.  Should not take up more than 250 MB of RAM
-8.  Commands should receive a response within 1 second
-9.  The system is not required to change the physical inventory
-10. The system should operate within a local network
-11. The data should be secured using a password
-12. Users should be able to get fluent with the syntax by their 10th usage
-13. The system should not provide functionality that breaks and local laws within a country it is distributed to
-14. The system should still be able to function without connection to a network
-15. The system should only be used by one user
+4.  Project should not cost any money.
+5.  Should work on 32-bit and 64-bit environments.
+6.  Should not take up more than 50 MB of disk space.
+7.  Should not take up more than 250 MB of RAM.
+8.  Commands should receive a response within 1 second.
+9.  The system is not required to change the physical inventory.
+10. The system should operate within a local network.
+11. The data should be secured using a password.
+12. Users should be able to get fluent with the syntax by their 10th usage.
+13. The system should not provide functionality that breaks and local laws within a country it is distributed to.
+14. The system should still be able to function without connection to a network.
+15. The system should only be used by one user.
+16. There should be enough RAM to store 100 states of the models for the Undo and Redo Commands.
+17. There should be enough RAM to store 100 states of history of commands the user has entered.
```

Comment:
I'm not really sure with myself either, but doesnt RAM depends on the user's device?

Reply:
Yeah it does, but this is NFR so you're stating that the user should have this

Reply:
Frm my understanding NFR needs to be something that is within our control, but in this case its more like dependent on the user and not within our control. Maybe can be rephrased like: Storing 100 states of undo and redo command models should take up less than ... Kb?? What do you think

Reply:
Ah I understand your concern, and yes the rephrasing is good. Will add it in